lib/util.js

```diff
@@ -489,3 +489,14 @@ exports.construct_hostname = function (region, account)
   }
   return host;
 };
+
+/**
+ * Returns if command is a PUT command
+ *
+ * @param sqlText the query command
+ * @returns {boolean}
+ */
+exports.isPutCommand = function (sqlText)
+{
+  return (sqlText.substring(0, 3) === "put" || sqlText.substring(0, 3) === "PUT");
```

Comment:
Alternatively you could capitalize `sqlText.substring(0, 3)` and just do one comparison

Reply:
That's a good idea, done